OutOfSchool/OutOfSchool.BusinessLogic/Services/ISocialGroupService.cs

```diff
@@ -28,7 +28,7 @@ public interface ISocialGroupService
     /// </summary>
     /// <param name="dto">SocialGroup entity to add.</param>
     /// <returns>A <see cref="Task{TResult}"/> representing the result of the asynchronous operation.</returns>
-    Task<SocialGroupDto> Create(SocialGroupCreate dto);
+    Task<SocialGroupCreate> Create(SocialGroupCreate dto);
```

Comment:
`SocialGroupCreate` should not be a response type of a service, as anything that is used to create - is an argument of a method.